core/Db/Schema/Mysql.php

```diff
@@ -102,6 +102,14 @@ public function getTablesCreateSql()
                             ) ENGINE=$engine DEFAULT CHARSET=utf8
             ",
 
+            'site_setting'    => "CREATE TABLE {$prefixTables}site_setting (
```

Comment:
Will it be serialized using `serialize()`? Unless it's really not practical it would be good to find alternatives instead of storing serialized strings: it is only readable by PHP, the format could change (e.g. even today a Piwik install cannot be migrated from PHP to HHVM because of that: not same serialization format), etc.